src/app/travel-app.component.ts

```diff
@@ -1,10 +1,13 @@
 import { Component } from "@angular/core";
+import { Router } from "@angular/router";
 
 @Component({
   selector: 'travel-app',
   templateUrl: 'travel-app.component.html'
 })
 
 export class TravelAppComponent {
+  // router?:any
```

Comment:
make sure you review each commit before committing for unused code and remove it.